DDS: Linux Audit Logs Updates (#20345)
DataDog/integrations-core: linux_audit_logs/README.md

    sudo systemctl restart auditd
    ```

> ### Setup Audit Rules (Optional)

Reviewer: ```suggestion
### Set up Audit Rules (optional)
```

> 1. Create/Edit the Audit Rules File

Reviewer 2: ```suggestion
1. Create or Edit the audit rules file.
```

> 1. Create/Edit the Audit Rules File

Reviewer: ```suggestion
1. Create or edit the Audit Rules file:
```

    ```shell
    sudo nano /etc/audit/rules.d/audit.rules
    ```

> 2. Configure the audit rules based on your requirements. For reference, check out [audit rulesets][9].

Reviewer: ```suggestion
2. Configure the audit rules based on your requirements. For reference, see [audit rulesets][9].
```

> 3. Reload Audit Rules

Reviewer: ```suggestion
3. Reload Audit Rules:
```

> 3. Reload Audit Rules

Reviewer 2: ```suggestion
3. Reload audit rules.
```

    ```shell

>     sudo augenrules --load

Reviewer 2: Just want to confirm that `augenrules` is the correct spelling.

Author: Yes, it's the correct spelling.

    ```

> 4. Verify Loaded Rules

Reviewer: ```suggestion
4. Verify loaded rules:
```

> 4. Verify Loaded Rules

Reviewer 2: ```suggestion
4. Verify loaded rules.
```
